templates/governance.yaml

```diff
@@ -273,12 +273,21 @@ Resources:
               - 'ec2:DescribeReservedInstancesOfferings'
               - 'ec2:ModifyReservedInstances'
               - 'ec2:PurchaseReservedInstancesOffering'
+              - 'elasticache:DescribeReservedCacheNodes'
+              - 'elasticache:DescribeReservedCacheNodesOfferings'
+              - 'elasticache:PurchaseReservedCacheNodesOffering'
+              - 'es:DescribeReservedInstanceOfferings'
+              - 'es:DescribeReservedInstances'
+              - 'es:PurchaseReservedInstanceOffering'
```

Comment:
I would also add:
[DescribeReservedElasticsearchInstanceOfferings](https://docs.aws.amazon.com/opensearch-service/latest/APIReference/API_DescribeReservedInstanceOfferings.html)
[DescribeReservedElasticsearchInstances](https://docs.aws.amazon.com/opensearch-service/latest/APIReference/API_DescribeReservedInstances.html)
[PurchaseReservedElasticsearchInstanceOffering](https://docs.aws.amazon.com/opensearch-service/latest/APIReference/API_PurchaseReservedInstanceOffering.html)

Reply:
But check with CCO team if its something they can leverage.

Reply:
Per [IAM actions docs](https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazonopensearchservice.html), those three actions are deprecated and replaced by the ones added here.